CHANGELOG.md

```diff
@@ -16,6 +16,12 @@ Semantic versioning in our case means:
   But, in the future we might change the configuration names / logic,
   change the client facing API, change code conventions signigicantly, etc.
 
+## 0.16.0
+
+### Features
+
+* Forbids getting first element of list by unpacking
```

Comment:
Done.

Maybe if the project have MD style rules, it can use some linter? I found rule that seems to fix bullet list style: https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md004---unordered-list-style

I may work on adding linter to the project.